src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.h

```diff
@@ -3,15 +3,26 @@
 
 #include "global.h"
 
+#define KIBAKO2_COLLECTIBLE_ID(thisx) ((thisx)->params & 0x3F)
```

Comment:
I think it would be nice if these macros took in "this" instead of "thisx" as an argument for two reasons. One, they are used multiple places in the code, so it actually makes the code quite a bit fewer characters to call a macro with an argument of `this` instead of `this->dyna.actor`, and two, these macros are "namespaced" to Kibako2, not to Actor, so it makes sense they should take a pointer to Kibako2, not Actor.
```suggestion
#define KIBAKO2_COLLECTIBLE_ID(this) (this->dyna.actor.params & 0x3F)
```